litellm/integrations/mlflow.py

```diff
@@ -0,0 +1,241 @@
+import json
+import threading
+from typing import Optional
+from litellm.integrations.custom_logger import CustomLogger
+from litellm.utils import verbose_logger
+
+import mlflow
```

Comment:
if this is a required import it should be moved inside the class, so it's only required when using mlflow